app/mci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -109,11 +109,11 @@ func healthCheckMatches(desiredHC, existingHC *compute.HealthCheck) bool {
 	return true
 }
 
-func (h *HealthCheckSyncer) desiredHealthCheck(lbName string, port sp.ServicePort) (compute.HealthCheck, error) {
+func (h *HealthCheckSyncer) desiredHealthCheck(lbName string, port ingressbs.ServicePort) (compute.HealthCheck, error) {
 	// Compute the desired health check.
 	hc := compute.HealthCheck{
 		Name:        h.namer.HealthCheckName(port.Port),
-		Description: fmt.Sprintf("Kubernetes L7 Loadbalancing health check for multi cluster ingress %s.", lbName),
+		Description: fmt.Sprintf("Health check for service %s as part of kubernetes multi cluster loadbalancer %s", port.Description(), lbName),
```

Comment:
jonathan was asking about "multicluster" spelling, and we said it was without a space.

Reply:
Removed the space